src/api/roles/client/client.py

```
@@ -215,55 +222,169 @@ def get_coach_availability_for_client(coach_id: int, db = Depends(get_session),
-
-    availabilities = db.exec(select(Availability).where(Availability.coach_availability_id == coach.coach_availability)).all()
-
-    return CoachAvailabilityResponse(coach_availabilities=availabilities)
+    coach = db.get(Coach, coach_id)
+    if coach is None:
+        raise HTTPException(404, detail="Coach not found")
+
+    coach_account = db.exec(
+        select(Account).where(
+            Account.coach_id == coach_id,
+            Account.is_active == True,
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `Account.is_active:` for truth checks

Replace with `Account.is_active`

(E712)